### ISSUE TITLE
Ignore accents

### ISSUE THREAD

**Issue by vwasteels** [jbr/jQuery.highlightRegex#22]
Hello,

Is there a way to ignore accents while highlighting ?

ex : ```la pâté c'est bon```
I would like to highlight `pâté`  when searching for `pate`

Thank you !!!

**Comment by julmot** [comment]
@vwasteels No, you need to include it in the RegExp yourself. However, if you're using a **keyword** highlighter, then you can use such options. An example would be [mark.js](https://markjs.io/). However, as mentioned it only supports keywords, not RegExp if you like to use the `diacritics` option.

**Comment by vwasteels** [comment]
@julmot  thanks for your quick answer, I will check this soon :)
